docs/UserGuide.adoc

```diff
@@ -651,6 +700,7 @@ image::emailSS2.PNG[width="400", align="center"]
 // end::email[]
 
 === Utility Features
+This section describes the commands you can use to utilize contactHeRo. +
```

Comment:
Change to:
This section describes the commands that will help you make better use of contactHeRo.